packages/vscode-extension/src/controls/QuickStart.tsx

```diff
@@ -36,6 +37,7 @@ export default class QuickStart extends React.Component<any, any> {
       m365Account: undefined,
       azureAccount: undefined,
       stepsDone: [false, false, false, false, false, false],
+      playFromStep: false,
```

Comment:
what is this property for?

Reply:
To distinguish whether user clicked from step primary button or the video play button